en/components/tree.md

```diff
@@ -0,0 +1,98 @@
+
+<img class="responsive-img" src="../images/tree_demo.png" srcset="../images/tree_demo@2x.png 2x" />
+
+## Tree Node State
+
+The Tree Node comes with the two primary states - Idle and Disabled. You can also choose from a variety of interaction states. In Figma this can be achieved by changing the value of the "State" property from the properties panel, while in Sketch it's achieved with `Symbol Overrides`.
+
+<img class="responsive-img" src="../images/tree-node_basic.png" srcset="../images/tree-node_basic@2x.png 2x" />
+<img class="responsive-img" src="../images/tree-node_disabled.png" srcset="../images/tree-node_disabled@2x.png 2x" />
+
+## Tree Presets
+
+In addition to the basic Tree component, you can also choose from three Tree Presets - Active, Cascading Selection and Multiple Selection. These can make building your design easier and faster. In Figma these are separate components, positioned on the same page as the Tree component and you can insert them from the "Assets" panel or the "Resources" panel. In Sketch these are under the Tree folder, in a separate group called "Presets". 
+
+<img class="responsive-img" src="../images/tree_presets_active.png" srcset="../images/tree_presets_active@2x.png 2x" />
+<img class="responsive-img" src="../images/tree_presets_cascading-selection.png" srcset="../images/tree_presets_cascading-selection@2x.png 2x" />
+<img class="responsive-img" src="../images/tree_presets_multiple-selection.png" srcset="../images/tree_presets_multiple-selection@2x.png 2x" />
+
+## Styling
+
+The Tree comes with styling flexibility through the available options for the Expand Icon, Checkbox and Text colors, as well as the colors used in different states of the Tree Node.
+
+<img class="responsive-img" src="../images/tree_styling.png" srcset="../images/tree_styling@2x.png 2x" />
+
+## Usage
+
+Make sure you switch the state of the Expand Icon to "Expanded" when you have expanded Тree Nodes. If some of the children (but not all) of a Tree Node are selected, make sure to switch the state of the Checkbox of the parent node to indeterminate state.
+
+| Do                                                                             | Don't                                                                              |
+| ------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------- |
+| <img class="responsive-img" src="../images/tree_do1.png" srcset="../images/tree_do1@2x.png 2x" /> | <img class="responsive-img" src="../images/tree_dont1.png" srcset="../images/tree_dont1@2x.png 2x" /> |
+| <img class="responsive-img" src="../images/tree_do2.png" srcset="../images/tree_do2@2x.png 2x" /> | <img class="responsive-img" src="../images/tree_dont2.png" srcset="../images/tree_dont2@2x.png 2x" /> |
+
+# Additional Resources
+
+Related topic:
+
+- [Checkbox](../components/checkbox.md)
+  <div class="divider--half"></div>
```

Comment:
Adding Icon.md as an additional resource? Also, in my commits, Svilen is removing the divider--half, but not sure about this